region/brinstar/pink/Dachora Room.json

```diff
@@ -231,6 +248,28 @@
         {"refill": ["Energy", "Missile", "Super"]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Blockade Moondance",
+      "notable": false,
+      "requires": [
+        "h_canUseMorphBombs",
+        {"or": [
+          {"and": [
+            "canMoondance",
+            "SpeedBooster"
```

Comment:
If you have SpeedBooster, then for doing down the Dachora pit there's no need to moondance, since you can just break the blocks with blue speed?